src/PharmacyProduct.java

```diff
@@ -0,0 +1,27 @@
+public class PharmacyProduct extends Product {
+    private String newType;
+    private double refund;
+    private String latinName;
+
+    public PharmacyProduct(String name, double price, String newType, double refund, String latinName) {
+        super(name, price);
+        this.newType = newType;
+        this.refund = 0.70;
```

Comment:
here you should set refund which is passed as a parameter 
`this.refund = refund;`